Core/blockerdata.js

```diff
@@ -32,8 +32,35 @@ duckduckgo_decodeBase64 = function(s) {
 
 var duckduckgoBlockerData = {
 
+    blockingEnabled: ${blockingEnabled},
 	disconnectme: ${disconnectme},
+    whitelist: ${whitelist},
     easylist: {},
     easylistPrivacy: {}
 
 }
+
+try {
+
+    ABPFilterParser.parse(function() {
+
+        var easylistData = duckduckgo_decodeBase64("${easylist_general}")
+        console.log("Easylist: " + easylistData.substring(0, 100))
```

Comment:
We should probably switch this logging off when in release mode, happy to capture that in a separate task though.

Reply:
can do - won't affect performance if the console is not open/attached though (having read the webkit code!)

Reply:
My logging concerns are more around privacy.

Reply:
Cool I've made a separate task in Asana https://app.asana.com/0/414709148257752/433059546511657